builtin/authority/authority_test.go

```diff
@@ -62,6 +62,15 @@ func TestAuthority(t *testing.T) {
 		{M(aut.Candidates(&big.Int{}, thor.InitialMaxBlockProposers)), M(
 			[]*Candidate{{p2, p2, thor.Bytes32{}, true}, {p3, p3, thor.Bytes32{}, true}}, nil,
 		)},
+		{M(aut.AllCandidates()), M([]*Candidate{
+			//{p1, p1, thor.Bytes32{}, false},
```

Comment:
Is this deletable or should be uncommented?

Reply:
Line deleted.